Library/UIViewController+URL.swift

```diff
@@ -0,0 +1,16 @@
+import SafariServices
+import UIKit
+
+public extension UIViewController {
+  static var supportedURLSchemes = ["http", "https"]
+
+  func goTo(url: URL, application: UIApplicationType = UIApplication.shared) {
```

Comment:
I think this dependency injection is totally fine - just wondering if we want to deviate from using `Environment` at this point in time?

Reply:
Can you elaborate a bit? Not sure where you going with `Environment`? This injection is mostly for testing purposes

Reply:
Oh, yes I mean we currently do all dependency injection through `AppEnvironment`/`Environment`, so this kind of injection (passing into a function) is fine, it's just that the rest of our injection hasn't been done that way. I'm more opening this up for discussion than anything else, I am not particularly opinionated about it.

Reply:
Oh right...I'll see what I can do and try to inject this through `AppEnvironment` instead